README.md

```diff
@@ -5,9 +5,6 @@ A pure Go implementation of https://github.com/signalapp/libsignal.
 This repository is meant to be broken down into different
 APIs similar to the source repository.
 
-Only libsignal-protocol is implemented at this time, 
```

Comment:
nit: it's still the case that only `libsignal-protocol` is implemented, so maybe just removing `, and it does not support groups at this time` would be best

Reply:
Yup, that’s right.